server/@types/express/index.d.ts

```diff
@@ -41,7 +40,6 @@ declare module 'express-session' {
     // Induction related forms held on the session
     inPrisonWorkForm: InPrisonWorkForm
     inPrisonTrainingForm: InPrisonTrainingForm
-    personalInterestsForm: PersonalInterestsForm
```

Comment:
We can remove the form type from the session as we no longer hold it on the session 👍